components/match2/commons/match_summary_base.lua

```diff
@@ -184,6 +185,22 @@ function Footer:addLinks(links)
 		end
 	end
 
+	Array.forEach(MATCH_LINK_PRIORITY, function(linkTypePrefix)
+		for linkType, link in Table.iter.pairsByPrefix(links, linkTypePrefix, {requireIndex=false}) do
+			processLink(linkType, link)
+		end
+	end)
+
+	local unorderedLinks = Table.filterByKey(links, function(key)
+		return Array.all(MATCH_LINK_PRIORITY, function (element)
+			return not String.startsWith(key, element)
```

Comment:
not sure this works 100% of the time
like if e.g. `steam` would be added to prio group while `steamalternative` and `steamtv` is not they still would be removed via this

Reply:
Hmm, could solve this with pattern matching, but was trying to avoid that.
Not sure whether this is an actual issue, links in matches should be fairly limited

Reply:
tend to agree, just wanted to mention it so we are aware of it